src/Goteo/Util/Form/FormFinder.php

```diff
@@ -36,20 +37,7 @@ public function getModel() {
         return $this->model;
     }
 
-    public function resolve($form) {
```

Comment:
Now we don't the the `resolve()` method since I'm always passing the full class name (E.g.: WhatEverForm::class)